administrator-manual/en/release_notes.rst

```diff
@@ -47,6 +56,10 @@ Major changes on 2020-11-XX
   :guilabel:`System > Users & Groups > [Account provider] > Edit provider`.
   See also :ref:`dedicated-service-account`.
 
+  To prevent errors during the upgrade to Nextcloud, the ``mail`` and ``theming`` have been disabled.
```

Comment:
Let's split the Nextcloud in two points: new/existing 

```suggestion
* To prevent errors during the upgrade of existing Nextcloud installations, the ``mail`` and ``theming`` applications have been disabled.
```